kuksa-client/kuksa_client/__main__.py

```diff
@@ -401,18 +406,11 @@ def do_subscribe(self, args):
         """Subscribe the value of a path"""
         if self.checkConnection():
 
-            logPath = pathlib.Path.cwd() / \
-                f"log_{args.Path.replace('/', '.')}_{args.attribute}_{str(time.time())}"
-            callback = functools.partial(self.subscribeCallback, logPath)
             resp = self.commThread.subscribe(
-                args.Path, callback, args.attribute)
+                args.Path, self.subscribeCallback, args.attribute)
             resJson = json.loads(resp)
             if "subscriptionId" in resJson:
                 self.subscribeIds.add(resJson["subscriptionId"])
-                logPath.touch()
-                print(f"Subscription log available at {logPath}")
-                print(
-                    f"Execute tail -f {logPath} on another Terminal instance")
```

Comment:
If we change in this method, should we not also change `do_subscribeMultiple`

Reply:
Yes, wasn't aware of that one.